docs/reference/sql/functions/index.asciidoc

```diff
@@ -137,6 +137,9 @@
 * <<sql-functions-geo>>
 ** <<sql-functions-geo-st-as-wkt>>
 ** <<sql-functions-geo-st-wkt-to-sql>>
+** <<sql-functions-geo-st-geometrytype>>
```

Comment:
I think this one has to go before wkt-to-sql? (they should be listed alphabetically for ease of browsing)